db/views/instances_v01.sql

```diff
@@ -0,0 +1,10 @@
+SELECT domain
+FROM accounts
+WHERE domain IS NOT NULL
+GROUP BY domain
+UNION
```

Comment:
The reason for the union queries is you may want servers to show up in the list even if you have no accounts from them as long as you've already blocked or allowed them, doubly so because the refactored `InstanceFilter` uses this view as the base query.